### PR TITLE
adding different colour for centroids and added  visualisation for devograph

### PR DESCRIPTION
**Pull request by sushmanthreddy**
Screenshot 2023-03-16 at 10 25 50 PM

here centroids are of different colour and membrane is of different colour added ,pr helps in visualisation for devograph

### CODE REVIEW

**Comment by devoworm** [comment]
Looks good! Requesting a check from @Mayukhdeb

File: devolearn/cell_membrane_segmentor/cell_membrane_segmentor.py [hunk 1]
```diff
@@ -115,7 +124,8 @@ def preprocess(self, image_grayscale_numpy):
         tensor = self.mini_transform(image_grayscale_numpy).unsqueeze(0).to(self.device)
         return tensor
 
-    def predict(self, image_path, pred_size = (350,250), centroid_mode = False):
+    def predict(self, image_path, pred_size = (350,250), centroid_mode = False, color_mode = False):
+        print("fdfdsfds")
```
Comment (Mayukhdeb) [inline devolearn/cell_membrane_segmentor/cell_membrane_segmentor.py]: I think this print statement is not needed. Please remove it 😅

**Comment by Mayukhdeb** [comment]
Thanks for prompt fix @sushmanthreddy 👍 
I would recommend you to also please use relevant commit messages for the changes you make. A commit message like `update {x}` is generally not very helpful for reviewers.

**Comment by sushmanthreddy** [comment]
> Thanks for prompt fix @sushmanthreddy 👍
> I would recommend you to also please use relevant commit messages for the changes you make. A commit message like `update {x}` is generally not very helpful for reviewers.

thanks for the review, from next pr onwards ,I will write relevant message.Anyways thanks for review

**Pull request merged**